docs/pages/dsHDMIIn/ds-hdmi-in-High-Level_TestSpec.md

```diff
@@ -416,3 +417,22 @@ Emulator will boot with the `ALLM` support information.
 ### Control Plane Requirement - Get and Set `EDID` to all `ALLM` Support
 
 Control plane connects/ disconnects the external devices. Validates the ALLM by analyzers.
+
+## Get `HDMI` Version
+
+|Test Functionality|Description|HAL APIs|L2|L3|Source|Sink|Control plane requirements|
+|------------------|-----------|--------|--|--|------|----|--------------------------|
+|Get `HDMI` Version|Get `HDMI` Version and Validate with the version from profile file|dsGetHdmiVersion()|`Y`|`N`|`N`|`Y`|`NA`|
+
+### Test Startup Requirement - Get `HDMI` Version
+
+- The test begins with the configured `HDMI` input port numbers.
+- Connection of the source device with the `HDMI` Input
```

Comment:
Remove this line, as the test setup does not require source device

Reply:
Done